src/stores/game.js

```diff
@@ -285,6 +287,13 @@ export const useGameStore = defineStore('game', {
     },
     opponentLeft() {
       this.players = this.players.filter((player) => player.pNum === this.myPNum);
+      
+      //set opponent as not ready (bug repair)
```

Comment:
Let's remove (bug repair) from the comment. Imo which code fixes which thing is useful to indicate in the PR commends (i.e. you can comment on a line in the PR like I'm doing now) to point out to the reviewer what's what, but the code comments themselves should just explain why we're doing what we're doing.
```suggestion
      // set opponent as not ready because they left
```